### DISCUSSION TITLE
Curse dispellable

### DISCUSSION THREAD

**Discussion by xSurrealz** [siweia/NDui#134] [Q&A] [answered]
Hello !

I wanted to know if it was possible to change the size of dispelable curses or aura like iceblock in the party group ?
And, its possible to block auto swap party to raid mode when i'm on a arena ?

Sorry for my english, nice UI !

I put a screen with my problem here : https://prnt.sc/wg9zx8

**Comment by siweia** [comment, marked as answer]
I do remember, there should be an option to scale that icon.
Swap your party to raid mode is setup by blizzard, not me.
